go/engine/signup.go

```diff
@@ -187,6 +188,9 @@ func (s *SignupEngine) registerDevice(a libkb.LoginContext, ctx *Context, device
 		Lks:        s.lks,
 		IsEldest:   true,
 	}
+	if s.arg.DeviceType == keybase1.DeviceType_MOBILE {
```

Comment:
Won't there be a bug if we introduce another type? We won't think to update this line.